client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -120,16 +121,20 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		return allDesigns;
 	};
 
+	const [ isLoadingVirtualThemesExperiment, virtualThemesExperiment ] = useExperiment(
+		'calypso_signup_design_picker_virtual_themes_v1'
+	);
+
 	const { data: allDesigns, isLoading: isLoadingDesigns } = useStarterDesignsQuery(
 		{
 			vertical_id: siteVerticalId,
 			intent,
 			seed: siteSlugOrId || undefined,
 			_locale: locale,
-			include_virtual_designs: isEnabled( 'virtual-themes/onboarding' ),
+			include_virtual_designs: virtualThemesExperiment?.variationName === 'treatment',
 		},
 		{
-			enabled: true,
+			enabled: ! isLoadingVirtualThemesExperiment,
```

Comment:
`enabled` indicates whether the API request to fetch the themes should run (see https://tanstack.com/query/v4/docs/react/reference/useQuery#:~:text=be%20undefined.-,enabled%3A%20boolean,-Set%20this%20to).

We should refrain from running the request until we know what experiment variation has been assigned the user to, since that determines a query param of such request (whether virtual themes are included or not).

Reply:
Thanks for explaining it! 💯